lib/aggregations/class-custom-date-range.php

```diff
@@ -72,16 +72,50 @@ function( $query_value ) {
 	 * @return array Array of DSL fragments to apply.
 	 */
 	public function filter(): array {
-		$date_range = $this->get_date_range(
-			$this->query_values[0] ?? '',
-			$this->query_values[1] ?? ''
-		);
+		$from = $this->query_values[0] ?? $this->get_default_date_range( 'from' );
+		$to   = $this->query_values[1] ?? $this->get_default_date_range( 'to' );
+
+		$date_range = $this->get_date_range( $from, $to );
 
 		return ! empty( $date_range )
 			? [ $this->dsl->range( 'post_date', $date_range ) ]
 			: [];
 	}
 
+	/**
+	 * Get the default date range.
+	 *
+	 * @param string $range The range to get the default date for.
+	 * @return string
+	 */
+	private function get_default_date_range( string $range ): string {
+		switch ( $range ) {
+			case 'from':
+				$datetime = gmdate( 'Y-m-d', 0 ); // Defaults to 1970-01-01.
```

Comment:
This works but I'm unsure if `gmdate` is the right function here. 🤔 

Does it matter it is in GMT since it will use `wp_timezone` later to create and format the date?! I guess not.